### ISSUE TITLE
Project snapshot generation not working when HTTP_FOLDER folder is not accessible

### ISSUE THREAD

**Issue by geourjoa** [SCENE-CE/mirador-multi-user#352]
On a dev clean install, project snapshot are not working (error in console, on fetching the manifest)

Problem was fixed by changing .env variable HTTP FOLDER ./upload to ./backend/upload

**Comment by Drilio** [comment]
Does manifest upload/create link are working ? 
Same questions for media upload

**Comment by Drilio** [comment]
clean install confirmation link is also broken : 
![image](https://github.com/user-attachments/assets/4938cab7-bbbe-4abb-beea-d5ce9e4f336b)

**Comment by geourjoa** [comment]
With the change of HTTP Folder :

Working 
- Upload manifest
- Link manifest
- create manifest 
- link image media
- upload image media

**Comment by Drilio** [comment]
seems like .env varible HOST is used there : https://github.com/SCENE-CE/mirador-multi-user/blob/255d118b2b1fcb9a1b290d700a7decfaedc9eb9a/docker-compose.yml#L57

**Comment by geourjoa** [comment]
Normally we don't send mail in dev mode 

Yes $HOST seem to be used in any case

**Comment by Drilio** [comment]
You're right on the mailer something went wrong I'll try to figure it out

**Comment by Drilio** [comment]
OK my bad I setted up SMTP variables, I'll add a comment to explain why it's important to let theme empty in dev mode

**Comment by Drilio** [comment]
ok updated .env.dev.sample there : https://github.com/SCENE-CE/mirador-multi-user/pull/353
